providers/implementations/include/prov/seeding.h

```diff
@@ -10,6 +10,38 @@
 #include "prov/provider_ctx.h"
 #include "crypto/rand_pool.h"
 
+/* Entropy quality for different operating systems */
+#if defined(OPENSSL_SYS_VOS)
+/*
+ * The entropy source repeatedly samples the real-time clock (RTC) to
+ * generate a sequence of unpredictable data.  The algorithm relies upon the
+ * uneven execution speed of the code (due to factors such as cache misses,
+ * interrupts, bus activity, and scheduling) and upon the rather large
+ * relative difference between the speed of the clock and the rate at which
+ * it can be read.
+ *
+ * As a precaution, we assume only 2 bits of entropy per byte.
+ */
+# define OS_ENTROPY_FACTOR  4
+#elif defined(__VMS)
+/*
+ * This number expresses how many bits of data contain 1 bit of entropy.
+ *
+ * For the moment, we assume about 0.05 entropy bits per data bit, or 1
+ * bit of entropy per 20 data bits.
```

Comment:
```suggestion
 * For the moment, we assume a worst case fallback scenario of 0.05 entropy
 * bits per data bit, or 1 bit of entropy per 20 data bits.
```

Reply:
FFS: I simply don't care about the wording here.
Update this PR if you care.

It is not even remotely interesting let alone important.

Reply:
> Update this PR if you care.

Check.  Will do.